examples/opf/clients/nyctaxi/README.md

```diff
@@ -0,0 +1,13 @@
+# CPU 
+
+This custom client is similar to hotgym client and demonstrates how to configure an OPF client with a Numenta's Cortical Learning Algorithm (CLA) to detect anomalies. The internal model is constantly generating predictions about the future. The model is configured to output an "anomaly score", a value between 0 and 1 which measures the discrepancy between the actual input and its predicted value.
+
+##Usage
+Simply run the nyctaxi_anomaly.py file to start the client.
+
+    ./nyctaxi_anomaly.py
+
+The parameters used to create the model are specified in `model_params.py`.
+
+##Output
```

Comment:
nit: space after heading indicator and newline after (before first line in the section)